docs/user/guides/git-integrations.rst

```diff
@@ -0,0 +1,204 @@
+How to connect your Git repository
+==================================
+
+In this guide, you will find the simple steps to integrating your Read the Docs project with GitHub, Bitbucket, GitLab, Gitea or any other Git provider that supports our generic API.
+
+To manually set up an integration, go to :guilabel:`Admin` > :guilabel:`Integrations` >  :guilabel:`Add integration`
```

Comment:
That makes total sense, yes :+1:

Reply:
Created a section "Additional integration". I was thinking that the word manual is already semi-occupied because the act of setting up the webhook is also kind of manual :)